### PR TITLE
Enhances the Numb quirk by expanding its effects

### PR DESCRIPTION
**Pull request by Acantharctia**
## About The Pull Request
Works on details overlooked on the original PR.

- Removes screams from damage.
- Removes screen overlay from damage.

## Why It's Good For The Game

Enhances the quirk by adding on to the bare-bones structure it had while also making it more mechanically unique.
## Changelog
:cl:
balance: Numb quirk now stops you from screaming and seeing the damage overlay when damaged. Better keep a close eye on your health.
/:cl:

### CODE REVIEW

File: code/datums/quirks/negative_quirks/numb.dm [hunk 1]
```diff
@@ -10,6 +10,8 @@
 
 /datum/quirk/numb/add(client/client_source)
 	quirk_holder.apply_status_effect(/datum/status_effect/grouped/screwy_hud/fake_healthy, type)
+	quirk_holder.add_traits(list(TRAIT_ANALGESIA))
```
Comment (SmArtKar) [inline code/datums/quirks/negative_quirks/numb.dm:13]:
```suggestion
	ADD_TRAIT(quirk_holder, TRAIT_ANALGESIA, QUIRK_TRAIT)
```

File: code/datums/quirks/negative_quirks/numb.dm [hunk 2]
```diff
@@ -10,6 +10,8 @@
...
 
 /datum/quirk/numb/remove(client/client_source)
 	quirk_holder.remove_status_effect(/datum/status_effect/grouped/screwy_hud/fake_healthy, type)
+	quirk_holder.remove_traits(list(TRAIT_ANALGESIA))
```
Comment (SmArtKar) [inline code/datums/quirks/negative_quirks/numb.dm:17]:
```suggestion
	REMOVE_TRAIT(quirk_holder, TRAIT_ANALGESIA, QUIRK_TRAIT)
```
A) you need to provide the source B) use ADD/REMOVE_TRAIT when working with a single trait

**Review by SmArtKar — CHANGES_REQUESTED** [review]
Also please keep in mind that TRAIT_ANALGESIA speeds up surgeries and prevents you from getting surgery mood debuffs, so its also a minor buff (although who cares about details as minor as that really (me))

**Comment by necromanceranne** [comment]
> Also please keep in mind that TRAIT_ANALGESIA speeds up surgeries and prevents you from getting surgery mood debuffs, so its also a minor buff (although who cares about details as minor as that really (me))

It's overall still a significant negative quirk, I've definitely had points where the only thing even hinting that I'm about to die is the red circle around the screen, and if that is removed, it absolutely would be near impossible to know when I'm about to die except trying to guess from damage slowdown. The circle was often not even enough for a hint, you really have to be paranoid about health.

Good change. I'm definitely going to convince myself I'm more healthy than I am with it and pay the price.

**Comment by Holinka4ever** [comment]
Isnt these quirk stops becoming pain in butt, after getting health analyzer?

**Review by SmArtKar — APPROVED** [review]

**Comment by SomeoneYouProbablyKnow** [comment]
> Isnt these quirk stops becoming pain in butt, after getting health analyzer?

Not really, even without one you can always examine yourself to see how fucked up you are. The danger comes from time where you don't have the time to do so.

**Comment by Holinka4ever** [comment]
> > Isnt these quirk stops becoming pain in butt, after getting health analyzer?
> 
> It's still something you need to keep in mind, it's possible to tank some hits and think you're okay when you're like 5 brute from crit

Okay, so what about health sensor?
Like, you can use it to tell where is you health threshold to make sure that you are not faint in a seconds.

**Comment by Acantharctia** [comment]
> > > Isnt these quirk stops becoming pain in butt, after getting health analyzer?
> > 
> > 
> > It's still something you need to keep in mind, it's possible to tank some hits and think you're okay when you're like 5 brute from crit
> 
> Okay, so what about health sensor? Like, you can use it to tell where is you health threshold to make sure that you are not faint in a seconds.

You'd have to go out of your way to get one, if you're not part of medical staff or a head.
I'm not sure what you're trying to say here. If you're trying to argue that it's so minor effect that it can be remedied by something, you could make the same point towards quirks like Nyctophobia or Mute. Both can be remedied and yet they're still fun and unique quirks that interact with the round differently.

**Pull request merged**